test/parallel/test-fs-symlink-dir.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const common = require('../common');
+
+if (!common.canCreateSymLink())
+  common.skip('insufficient privileges');
+
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs');
+
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+
+const testDirA = path.join(tmpdir.path, 'a');
+const testDirB = path.join(tmpdir.path, 'b');
+const testDirC = path.join(tmpdir.path, 'c');
+fs.mkdirSync(testDirA);
+
+// Create a symlink pointing to a directory.
+
```

Comment:
I think you need to `process.chdir` other wise you are creating the links in the repo root